content/applications/marketing/events/event_essentials.rst

```diff
@@ -2,283 +2,384 @@
-invite: :guilabel:`Send`, :guilabel:`Schedule`, and :guilabel:`Test`.
+To learn more about activities in Odoo, refer to the :doc:`activities documentation
+<../../essentials/activities>`.
 
-The :guilabel:`Send` option sends the invite right away. The :guilabel:`Schedule` option reveals a
-pop-up, in which a scheduled date/time can be selected for the email to be sent. The
-:guilabel:`Test` option reveals a :guilabel:`Test Mailing` pop-up, where specific recipient email
-addresses can be entered for Odoo to send them the current version of the mailing for review before
-officially sending it out to prospective event attendees.
+Settings
+========
 
-If the selected :guilabel:`Mailing Type` is :guilabel:`SMS`, there are four options to send the
-invite: :guilabel:`Put in Queue`, :guilabel:`Send Now`, :guilabel:`Schedule`, and :guilabel:`Test`.
+To access the event settings and feature options in Odoo *Events*, navigate to
+:menuselection:`Events app --> Configuration --> Settings`. From here, tick the checkboxes beside
+the desired settings and/or features, and click :guilabel:`Save` to activate them.
 
-The :guilabel:`Put in Queue` option schedules an SMS message to all recipients (that match the
-designated rules, if any) in the near future. Clicking :guilabel:`Put in Queue` reveals a pop-up,
-requiring confirmation. When confirmed, a blue banner appears on the event invite template form,
-indicating that the SMS will be sent later that day.
+Events section
+--------------
+
+In the :guilabel:`Events` section of the :guilabel:`Settings` page, there are selectable features
+that can be enabled to add various elements to events created with the Odoo *Events* application.
+
+.. image:: event_essentials/settings-events-section.png
+   :align: center
+   :alt: The Events section on the Odoo Events settings page in the Odoo Events application.
+
+The :guilabel:`Schedule & Tracks` feature allows users to manage and publish a schedule with tracks
+for events. *Tracks* is a catch-all term that refers to talks, lectures, demostrations,
+presentations, and other similar elements that users may choose to include as part of an event.
+
+When the :guilabel:`Schedule & Tracks` feature is enabled, two additional fields appear beneath it:
+:guilabel:`Live Broadcast` and :guilabel:`Event Gamification`.
+
+The :guilabel:`Live Broadcast` feature lets users air trakcs online, via a *YouTube* integration.
+
+The :guilabel:`Event Gamification` feature lets users share a quiz after any event track, in order
+for attendees to gauge how much they learned from the track they just saw/heard.
+
+Next, is the :guilabel:`Online Exhibitors` feature. This feature allows uers to display sponsors and
+exhibitors on event pages, which can serve as a valuable incentive to encourage partners and
+businesses to participate in the event.
+
+The :guilabel:`Jitsi Server Domain` field represents an external conferencing service that is
```

Comment:
I have never heard of this before- not sure if there is other documentation (internal or external) to link to this? Just an idea.

Reply:
I actually wasn't familiar with it, either. Had to do some research just to figure it out. I also searched for any existing on doc about it, but came up empty. If anything gets created that mentions it, though, I'll be sure to toss it in 👍